extractor/src/test/java/org/schabi/newpipe/extractor/services/soundcloud/SoundcloudPlaylistExtractorTest.java

```diff
@@ -70,7 +73,9 @@ public void testRelatedItems() throws Exception {
         }
 
         @Test
-        public void testMoreRelatedItems() {
+        public void testMoreRelatedItems() throws Exception {
+            defaultTestMoreItems(extractor);
```

Comment:
Why did you change this? This test was to check that this playlist indeed doesn't have a next page of items.

Reply:
Now the first page of items does not contain the 10 items, but only 3 of them

Reply:
Then the lines below should be deleted, however it would be better to change it to a playlist with less than 3 streams.

Reply:
Ok, will do